src/SupertypeLogger.ts

```diff
@@ -63,6 +63,7 @@ export class SupertypeLogger {
      */
     setLogger(loggerFunction: LoggerFunction) {
         this.sendToLog = loggerFunction;
+        SupertypeLogger.prototype.sendToLog = loggerFunction;
```

Comment:
Would it be better if instead of overriding `SupertypeLogger.prototype.sendToLog` directly here, we directly change the implementation of `private sendToLog(logLevel, logObject, ...rawLogData)` ?
```private sendToLog(logLevel, logObject, ...rawLogData) {
        console.log(this.prettyPrint(logLevel, logObject));     // eslint-disable-line no-console
    }```

Reply:
🤔 - great suggestion - so then we can just implement the PII logger by default within Sueprtype? @yrw1027

Reply:
I think one immediate issue we'll have with that is the hit module logger is a private logger

Reply:
scratch this :) - closing this solution